### DISCUSSION TITLE
이동 컨트롤러 <Zepeto Screen Touchpad>의 Movement Range를 수정할 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#583] [5. Q&A]
해당 스크립트의 원리는 모르겠지만
해당 Movement Range를 수정하는 것으로 컨트롤러의 걷기 제한이 가능하다는 것을 발견해서, 해당 수치를 조절하고 싶습니다.

![image](https://user-images.githubusercontent.com/5895396/161674683-69aff0e9-ba9a-4f2c-967f-0175bf9d4c93.png)

다만 열리지 않는 데다가, 해당 스크립트를 받아와도 movement 수치가 변경이 불가능해서 문의 드립니다.

**Comment by pumfit** [comment]
ZepetoPlayers의 virtual controller를 아래와 같이 로컬 경로로 복사해준 뒤 해당되는 프리팹안에서 movement  값을 수정해주시면 됩니다.

1.ZepetoPlayers에서 변경할 프리팹을 선택해 해당 경로로 이동합니다.
![image](https://user-images.githubusercontent.com/46295539/161680292-c98167ea-9923-4a47-8927-4902fa3c34e5.png)
2.프리팹을 드래그앤 드랍하여 로컬 경로로 복사해줍니다.
![image](https://user-images.githubusercontent.com/46295539/161680602-e0b139a3-99b4-45bd-9d48-83196ec2d69a.png)
3.복사한 프리팹에 movement 수치를 변경합니다. (그리고 복사한 프리팹 이름도 여기서 변경해주었습니다)
![image](https://user-images.githubusercontent.com/46295539/161680939-e4b75dce-5176-43dd-ae46-35f578c56add.png)
4. 복사한 커스텀 컨트롤러 프리팹을 ZepetoPlayers에 넣어주고 실행하면 수치가 반영된 모습을 볼 수 있습니다.
![image](https://user-images.githubusercontent.com/46295539/161681188-385ca757-4c14-4c5f-a992-53a308ace402.png)

**Reply by ryusiin** [reply]
감사합니다:> 확인했습니다
